crawl-ref/source/god-abil.cc

```diff
@@ -1747,7 +1747,7 @@ void yred_make_bound_soul(monster* mon, bool force_hostile)
     {
         if (orig.heads() < mon->props[OLD_HEADS_KEY].get_int())
         {
-            mprf("Yredelemnul restores the hydra's soul to it's natural glory!");
+            mprf("Yredelemnul restores the hydra's soul to its natural glory!");
```

Comment:
Since this message is a string without any parameters, it should use mpr() instead of mprf().